util/scripts/sync_tas/sync_pi_status_from_tas.py

```diff
@@ -0,0 +1,126 @@
+import os
+import sys
+import logging
+import django
+
+# loading up django so we can use models for queries/updates
+sys.path.append('/project') 
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "chameleon.settings")
+django.setup()
+logger = logging.getLogger('util.scripts.sync_tas.sync_pi_status')
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from pytas.http import TASClient
+from chameleon.models import PIEligibility
+from django.utils import timezone
+from datetime import datetime
+import argparse
+
+def main(argv=None):
+    if argv is None:
+        argv = sys.argv
+
+    parser = argparse.ArgumentParser(\
+        description='Import PI eligibility status from TAS')
+    parser.add_argument('--dryrun', type=bool, help=\
+        'Simulate import, no changes applied', default=False)
+    parser.add_argument('--users', type=str, help=\
+        'Synchronize a specific user or set of users using a csv list of usernames', default=None)
+    parser.add_argument('--seteligible', type=bool, help=\
+        'Used with --users flag to set all provided users as PI Eligible', default=False)
+    args = parser.parse_args(argv[1:])
+
+    started = datetime.now()
+    logger.info('Starting... {0}'.format(started))
+    usermodel = get_user_model()
+    if args.users:
+        usernames = args.users.split(',')
+        users = usermodel.objects.filter(username__in=usernames)
+    else:
+        users = usermodel.objects.all()
+
+    if args.users and args.seteligible:
+        updated = set_all_pi_status(args.dryrun, users, "Eligible")
+    else:
+        updated = import_pi_status(args.dryrun, users, TASClient())
```

Comment:
The `set_eligible` flag is never used on this function, should that code path inside import_pi_status be removed?

Reply:
yes, good catch, thanks!